opencensus-shim/src/main/java/io/opentelemetry/opencensusshim/internal/metrics/MetricAdapter.java

```diff
@@ -370,12 +370,13 @@ static long mapTimestamp(@Nullable Timestamp time) {
     return TimeUnit.SECONDS.toNanos(time.getSeconds()) + time.getNanos();
   }
 
-  private static List<ValueAtPercentile> mapValueAtPercentiles(
+  private static List<ValueAtQuantile> mapValueAtPercentiles(
       List<Summary.Snapshot.ValueAtPercentile> valueAtPercentiles) {
-    List<ValueAtPercentile> result = new ArrayList<>(valueAtPercentiles.size());
+    List<ValueAtQuantile> result = new ArrayList<>(valueAtPercentiles.size());
     for (Summary.Snapshot.ValueAtPercentile censusValue : valueAtPercentiles) {
       result.add(
-          ImmutableValueAtPercentile.create(censusValue.getPercentile(), censusValue.getValue()));
+          ImmutableValueAtQuantile.create(
+              censusValue.getPercentile() / 100.0, censusValue.getValue()));
```

Comment:
This is the only logic change

Reply:
this does end up being a breaking change for any OC shim users, I guess. Do we have any?